### PR TITLE
Docs: add/update frontmatter descriptions for Dashboards pages

### PR DESCRIPTION
**Pull request by imatwawana**
Adding concise descriptions for each page in the Dashboards documentation
This PR must be merged by December 15th

### CODE REVIEW

File: docs/sources/dashboards/dashboard-public/index.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ labels:
     - enterprise
     - oss
 title: Public dashboards
+desription: Share your Grafana dashboards with anyone using public dashboards
```
Comment (imatwawana) [inline docs/sources/dashboards/dashboard-public/index.md:11]:
```suggestion
description: Share your Grafana dashboards with anyone using public dashboards
```

**Review by dprokop — APPROVED** [review]

File: docs/sources/dashboards/dashboard-public/index.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ labels:
     - enterprise
     - oss
 title: Public dashboards
+description: Share your Grafana dashboards with anyone using public dashboards
```
Comment (lucychen-grafana) [inline docs/sources/dashboards/dashboard-public/index.md:11]: "using public dashboards" might be mistakenly interpreted as describing the "anyone". Maybe something like this instead:
"Make your Grafana dashboards public and share it with anyone"

Comment (imatwawana) [reply docs/sources/dashboards/dashboard-public/index.md:11]: Ah makes sense!

Comment (imatwawana) [inline docs/sources/dashboards/dashboard-public/index.md:11]:
```suggestion
description: Make your Grafana dashboards public and share them with anyone
```

**Pull request merged**